### ISSUE TITLE
`wezterm-app`: Hashes don't match

### ISSUE THREAD

**Issue by wizard-28** [pacstall/pacstall-programs#2268]
```
~ via 🐍 v3.10.6
❯ pacstall -PI wezterm-app
[*] WARNING: Prompts are disabled
Do you want to view/edit the pacscript [y/N] N
[+] INFO: Sourcing pacscript
[+] INFO: Retrieving packages
Initializing download: https://github.com/wez/wezterm/releases/download/20221119-145034-49b9839f/WezTerm-20221119-145034-49b9839f-Ubuntu18.04.AppImage
File size: 43.2746 Megabyte(s) (45376704 bytes)
Opening output file WezTerm-20221119-145034-49b9839f-Ubuntu18.04.AppImage
Starting download

Connection 0 finished
Connection 2 finished
Connection 3 finished
Connection 1 finished
Connection 0 finished
[100%] [................................................................] [ 184.5KB/s] [00:00]

Downloaded 43.2746 Megabyte(s) in 4:00 minute(s). (184.52 KB/s)
[!] ERROR: Hashes do not match
[+] INFO: Cleaning up
```

@smokeythemonkey

**Comment by smokeythemonkey** [comment]
Will fix it later today

**Comment by smokeythemonkey** [comment]
https://github.com/pacstall/pacstall-programs/pull/2272

**Comment by Henryws** [comment]
Fixed in #2272.
